cli/azd/cmd/init.go

```diff
@@ -358,17 +366,15 @@ const (
 	initUnknown = iota
 	initFromApp
 	initAppTemplate
-	initProject
 	initEnvironment
 )
 
 func promptInitType(console input.Console, ctx context.Context) (initType, error) {
 	selection, err := console.Select(ctx, input.ConsoleOptions{
 		Message: "How do you want to initialize your app?",
 		Options: []string{
-			"Use code in the current directory",
+			"Use code in the current directory", // This now covers minimal project creation too
```

Comment:
```suggestion
			"Scan current directory", // This now covers minimal project creation too
```

This was one change we briefly talked about. "Use code in current directory" sounds like it doesn't work if you don't have code in the file. "Scan current directory" is accurate, and hopefully feels like your directory can be scanned even if it's empty.

Reply:
Thank you! I forgot about that one